src/App.js

```diff
@@ -13,7 +13,7 @@ const App = () => {
     const [data, setData] = useState(stays);
     const [filterLocation, setFilterLocation] = useState('')
 
-
+    // I feel there no point for this since you didn't check for anything
```

Comment:
Okay, I think I now know why you wanted another copy of the data, I guess so that you'll still have the original data